WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -25,12 +29,20 @@ public partial class AdditionalInformation : Window
         private List<TextBox> textBoxes = new List<TextBox>();
         private TextBox name = new TextBox();
         private Grid DynamicGrid = new Grid();
```

Comment:
You must not create a wpf component directly in the code

Reply:
Grid is now in xaml, but TextBoxes are created dynamically based on MapObject.Description.